Example/android/app/build.gradle

```diff
@@ -52,16 +51,6 @@ react {
     // hermesFlags = ["-O", "-output-source-map"]
 }
 
-project.ext["isReanimatedExampleApp"] = true
```

Comment:
Is it safe to remove this line?

Reply:
Hmm I didn't spot that it has been removed, but the app seems to work correctly without it so 🤷‍♂️

Reply:
The purpose of this flag is to resolve react-native directory to the one in example app's node_modules so this was added on purpose. We will need a stronger confirmation than "it works now".

Reply:
Ok so I'll revert it